dango/testing/tests/oracle.rs

```diff
@@ -20,7 +27,7 @@ const VAA_2: &str = "UE5BVQEAAAADuAEAAAAEDQBLJRnF435tmWmnpCautCMOcWFhH0neObVk2iw
 
 pub const WBTC_USD_ID: &str = "0xc9d8b075a5c69303365ae23633d4e085199bf5c520a3b90fed1322a0342ffc33";
 pub const ETH_USD_ID: &str = "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace";
-pub const USDC_USD_ID: &str = "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace";
+pub const USDC_USD_ID: &str = "0xeaa020c61cc479712813461ce153894a96a6c00b21ed0cfc2798d1f9a9e9c94a";
 pub const SOL_USD_ID: &str = "0xef0d8b6fda2ceba41da15d4095d1da392a0d2f8ed0c6c7bc0f4cfac8c280b56d";
 pub const ATOM_USD_ID: &str = "0xb00b60f88b03a6a625a8d1c048c3f66653edf217439983d037e7222c4e612819";
 pub const BNB_USD_ID: &str = "0x2f95862b045670cd22bee3114c39763a4a08beeb663b145d283c31d7d1101c4f";
```

Comment:
Are these the real pyth IDs for pyth mainnet? If so, we can make these constants in `dango_types::oracle` and import them in the tests.

Also, I think they can be defined as `PythId` consts instead of `&str` using `hex-literal`:

```rust
use hex_literal::hex;

const WBTC_USDC_ID: PythId = PythId::from_inner(hex!("0x..."));
```